src/main/java/Duke.java

```diff
@@ -19,26 +25,76 @@ public static void main(String[] args) {
+//                    "Added: " +
+//                    input +
+//                    "\n____________________________________________________________\n";
+//            System.out.println(output);
+//            input = sc.nextLine();
+//        }
+//        for(int i = 0; i < numberOfInputs; i++){
+//            System.out.print(i + 1);
+//            System.out.println(". " + myList[i]);
+//        }
+//        Level-2
+
+
+        Task[] myList;
+        myList = new Task[1000];
         Scanner sc = new Scanner(System.in);
         String input = sc.nextLine();
         int numberOfInputs = 0;
-        while(!input.equals("list")){
-            myList[numberOfInputs] = input;
-            numberOfInputs++;
-            String output = "____________________________________________________________\n " +
+        while(!input.equals("bye")) {
+            String words[] = input.split(" ");
+            if (input.equals("list")) {
+                for (int i = 0; i < numberOfInputs; i++) {
+                    System.out.print(i + 1);
+                    System.out.print(".");
+                    System.out.print(myList[i].getStatusIcon());
+                    System.out.println(myList[i].description);
+                }
+            }
+            else if (words[0].equals("mark")) {//pending check if second character is number AND smaller than numberOfInputs
+                int index = Integer.parseInt(words[1]);
+                myList[index-1].markAsDone();
+                System.out.println("____________________________________________________________\n" +
+                        "     Nice! I've marked this task as done:\n" +
+                        myList[index-1].getStatusIcon() + " " + myList[index-1].description +
+                        "    ____________________________________________________________");
+            }
+            else if (words[0].equals("unmark")) {//pending check if second character is number AND smaller than numberOfInputs
+                int index = Integer.parseInt(words[1]);
+                myList[index-1].markAsNotDone();
+                System.out.println("____________________________________________________________\n" +
+                        "     Alright! I've marked this task as not done yet:\n" +
+                        myList[index-1].getStatusIcon() + " " + myList[index-1].description +
+                        "    ____________________________________________________________");
+            } else {
+                myList[numberOfInputs] = new Task(input);
+                numberOfInputs++;
+                String output = "____________________________________________________________\n " +
                     "Added: " +
                     input +
                     "\n____________________________________________________________\n";
             System.out.println(output);
+            }
```

Comment:
A very long conditional loop, could consider using switch statement and refactoring the code to improve user readability.